events/events-team/registration/README.md

```diff
@@ -45,7 +45,8 @@ A successful Registration lead should have the following qualities:
 
 ## Shadow to the Registration Lead Role
 This role can include one or more shadows. The Registration Lead can delegate/assign the tasks to the shadows. The expectation from a shadow is to lead one of the events-team roles in an upcoming summit. The shadows to the Registration Lead are responsible for handling all Registration lead related activities in the absence of the Lead.
-
+After shadows are selected and confirmed, add all lead and shadows' emails to the summit-team@kubernetes.io group in https://github.com/kubernetes/k8s.io/blob/main/groups/sig-contributor-experience/groups.yaml under 
+.groups.[?(@.email-id=='summit-team@kubernetes.io')].members
```

Comment:
Should this be something like this, instead?
```suggestion
`.groups.[?(@.email-id=='summit-team@kubernetes.io')].members`.
```
I'm not sure whether the query language here will make sense to all possible leads (as some folks may be coming from a non-code background versus perhaps linking to the line in the file. But not a blocker :)

Reply:
I added the above suggestion and added a descriptive version of the query
@nimbinatus , ptal

Reply:
Looks good!